python/utilities/ev_whitelist/create_ev_whitelist_zip.py

```diff
@@ -21,13 +21,13 @@
 TRUNCATED_HASH_LENGTH = 8
 GOLOMB_M_PARAMETER = 2 ** 47
 NUM_FETCHING_PROCESSES = 40
-LOGS_LIST = [
-    "https://ct.googleapis.com/pilot",
-    "https://ct.googleapis.com/aviator",
-    "https://ct1.digicert-ct.com/log",
-    "https://log.certly.io",
-    "https://ct.izenpe.com",
-]
+LOGS_LIST = {
+    "https://ct.googleapis.com/pilot":6082932,
```

Comment:
re: reducing the whitelist size.

AFAICT this can only happen by 'natural wastage' ie. no longer including certs in the whitelist that once were. eg. if they expire.